stackdriver-core/.rubocop.yml

```diff
@@ -1,41 +1,62 @@
+inherit_gem:
+  google-style: google-style.yml
+
 AllCops:
   Exclude:
     - "stackdriver-core.gemspec"
     - "Rakefile"
     - "test/**/*"
-  TargetRubyVersion: 2.2
 
 Documentation:
   Enabled: false
 
-Style/StringLiterals:
-  EnforcedStyle: double_quotes
-Style/MethodDefParentheses:
-  EnforcedStyle: require_no_parentheses
-Style/ClassVars:
-  Exclude:
-    - "lib/google/cloud/configuration.rb"
-Metrics/CyclomaticComplexity:
-  Max: 10
-Metrics/PerceivedComplexity:
-  Max: 10
-Metrics/AbcSize:
-  Max: 25
-Metrics/ClassLength:
+Layout/AlignHash:
   Enabled: false
-Metrics/MethodLength:
-  Max: 20
 Metrics/ModuleLength:
   Enabled: false
-Metrics/ParameterLists:
-  Enabled: false
-Layout/EmptyLineAfterGuardClause:
-  Enabled: false
-Layout/EmptyLines: # for the extra line between copyright and code
+Metrics/ClassLength:
   Enabled: false
 Naming/AccessorMethodName:
-  Exclude:
-    - "lib/stackdriver/core/async_actor.rb"
+  Enabled: false
 Naming/FileName:
   Exclude:
     - "lib/stackdriver-core.rb"
+Style/EmptyMethod:
+  Enabled: false
+Style/IfUnlessModifier:
+  Enabled: false
+Style/MethodCallWithArgsParentheses:
+  Enabled: false
+Style/NumericPredicate:
+  Enabled: false
+Style/SafeNavigation:
+  Enabled: false
+
+# Style/StringLiterals:
```

Comment:
Can these commented out rules be removed?

Reply:
Whoops, missed those. Thanks!